keyboards/ergoinu/config.h

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+Copyright 2018 Takuya Urakawa<twitter:@hsgw>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+#include <serial_config.h>
+
+/* USB Device descriptor parameter
+   VID & PID are lisenced from microchip sublisence program, Don't use other project! */
+#define VENDOR_ID       0x04D8
+#define PRODUCT_ID      0xEE60
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Dm9Records
+#define PRODUCT         ergoinu
+#define DESCRIPTION     An (Not Portable But Small) Ergonomic split keyboard
+
+
+#define PREVENT_STUCK_MODIFIERS
+#define TAPPING_FORCE_HOLD
+#define TAPPING_TERM 100
+
+#define USE_SERIAL
+
+/* Select hand configuration */
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_ROW_PINS { F6, F7, B1, B3, B2 }
+
+// wiring of each half
+#define MATRIX_COLS 7
+#define MATRIX_COL_PINS { B4, E6, D7, C6, D4, F5, F4 }
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* number of backlight levels */
+// #define BACKLIGHT_LEVELS 3
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+//#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+//#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
+
+/* ws2812 RGB LED */
+#define RGB_DI_PIN B6
+#define RGBLIGHT_TIMER
+
+#define ws2812_PORTREG  PORTB
```

Comment:
These two lines aren't needed, I believe.

Reply:
I delete unused lines..

In this keyboard, the states of the left and right matrices are synchronized like HELIX, and serial sync feature is not implemented in "split common" .
So I want to use my codes.